zabbix/docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "3.7"
+
+services:
+
+  # https://hub.docker.com/r/zabbix/zabbix-web-nginx-pgsql
+  zabbix-frontend:
+    # docker run --name some-zabbix-web-apache-mysql
+    image: zabbix/zabbix-web-nginx-pgsql:7.0.3-alpine@sha256:f541682b9cb5a7543153235c889de2f090b30b91ed93d0596409d624b29592a6
+    hostname: zabbix-frontend
+    restart: on-failure
+    environment:
+      DB_SERVER_HOST: zabbix-database
+      POSTGRES_USER: zabbix
+      POSTGRES_PASSWORD: zabbix
+      ZBX_SERVER_HOST: zabbix-server
+      PHP_TZ: UTC
+    ports:
+      - "10052:8080"
```

Comment:
We can add our app_proxy container as the first one in this compose file and point it towards zabbix_zabbix-frontend_1 port 8080. We can then remove this port mapping. See Radarr as an example:

https://github.com/getumbrel/umbrel-apps/blob/master/radarr/docker-compose.yml#L4-L8

Reply:
I can apply this update, but then I think I must highlight the fact that there are other apps (like pi-hole) not following this convention. If we need to apply this update here then it must also be implemented in other containers for consistency.

<img width="814" alt="Screenshot 2024-09-13 at 19 13 00" src="https://github.com/user-attachments/assets/5ab85a32-c317-437b-a902-e1c471dfc4a5">

Reply:
Fixed for this new App

Reply:
https://github.com/getumbrel/umbrel-apps/issues/1463

Reply:
Discussed and closed https://github.com/getumbrel/umbrel-apps/issues/1463